data/azure/sql_service_tier_types.json

```diff
@@ -123,56 +123,36 @@
       "down": null
     },
     "GP_Gen5_4": {
-      "up": "6",
-      "down": "2"
-    },
-    "GP_Gen5_6": {
       "up": "8",
-      "down": "4"
+      "down": null
```

Comment:
Should this be null? Wouldn't this downsize to 2 e.g. GP_Gen5_2?

Reply:
I was going by the docs noted in the ticket that does not show a 2 instance.
BK noted that we should not use the docs at the URL I posted and we should go by these docs.
https://learn.microsoft.com/en-us/azure/azure-sql/database/resource-limits-vcore-single-databases?view=azuresql

It seems the issue noted is one specific to pepsi and not an issue with the SQL mapping file.

Reply:
@mburmeig Then that size shouldn't be listed in the document at all.

Let me know if you've confirmed that this change is unneeded and we can delete this PR.